packages/react-core-notifications/src/__tests__/__snapshots__/index.spec.ts.snap

```diff
@@ -2,6 +2,10 @@
 
 exports[`@aws-amplify/ui-react-core-notifications exports should match snapshot 1`] = `
```

Comment:
Where is the test file that created this?

Reply:
Included in PR to add package boilerplate
https://github.com/aws-amplify/amplify-ui/pull/3747/files#diff-ec87130187bf9355dab1a52bf1950550be5768313a68bb64fa90185ea6868097

Reply:
That makes sense 😄